dags/include/full_load_all_skus.py

```diff
@@ -143,55 +135,53 @@ def run_full_load():
            else pcpm2.code
            end as char) as base_code
     , case when coalesce(pcpm.code,pcpm2.code) like 'm-%' then true else false end  is_manual
-    
+
        ,pcp.created
-        
+
         from akeneo.pim_catalog_product                     pcp
                 left join akeneo.pim_catalog_product_model pcpm
                             on pcpm.id = pcp.product_model_id
                 left join akeneo.pim_catalog_product_model pcpm2
                             on pcpm.parent_id = pcpm2.id
-    
-    
+
+
                 left join (select max(sku)                   as sku,
                                 MAX(base_unit_content)     as base_unit_content,
                                 MAX(base_unit_content_uom) as base_unit_content_uom,
                                 MAX(no_of_base_units)         no_of_base_units,
                                 MAX(gtin)                  as gtin,
-    
+
                                 MAX(kollex_active)         as kollex_active,
                                 MAX(manufacturer)          as manufacturer,
-    
-    
+
+
                                 MAX(refund_value)          as refund_value,
-    
-    
+
+
                                 MAX(sales_unit_pkgg)       as sales_unit_pkgg,
                                 MAX(name)                  as name,
                                 MAX(active)                as active,
                                 MAX(category_code)            category_code,
-    
+
                                 MAX(direct_shop_release)      direct_shop_release
-    
+
                             from gfghdata.product
                             where sku is not null
                             group by sku) as gfghproduct on gfghproduct.sku = pcp.identifier
                 left join akeneo.pim_catalog_family_translation pcft on pcp.family_id = pcft.foreign_key
-                where identifier not in ({pg_identifiers})
             """,
         con=mysql_engine,
         chunksize=chunk_size,
     )
 
-    # connection = pg_engine.connect()
+    connection = pg_engine.connect()
 
-    # connection.execute(
-    #     f"drop table if exists {pg_raw_schema}.{pg_tables_to_use};"
-    # )
+    connection.execute(
```

Comment:
if merchant_active count is different from current_active_merchant_count (stored in database) we need to delete whole table because there is new merchant columns and can't only append.